pkgs/kswitch/kswitch.sh

```diff
@@ -506,16 +507,16 @@ while (("$#")); do
         exit 1
         ;;
     *)
-        [ "$zone" != "" ] && (log-error "too much arguments" && usage && exit 1)
+        [ "$zone" != "" ] && { log-error "too much arguments"; usage; exit 1; }
         zone=$1
         shift
         ;;
     esac
 done
 
 exitNow=0
-[ "${cleanCache}" -eq 1 ] && clean_cache && exitNow=1
-[ "${forceUnlock}" -eq 1 ] && force_unlock && exitNow=1
+[ "${cleanCache}" -eq 1 ] && { clean_cache; exitNow=1; }
+[ "${forceUnlock}" -eq 1 ] && { force_unlock; exitNow=1; }
 [ "${exitNow}" -eq 1 ] && exit 0
```

Comment:
```suggestion
[ "${exitNow}" -eq 1 ] && exit 1
```

Reply:
I don't think `--clean-cache` & `--force-unlock` should exit with error when they applied correctly

Reply:
Right, my bad